spark/v2.4/spark/src/test/java/org/apache/iceberg/spark/source/TestIcebergSourceTablesBase.java

```diff
@@ -898,7 +898,7 @@ public void testSnapshotsTable() {
                     ImmutableMap.of(
                         "added-records", "1",
                         "added-data-files", "1",
-                        "changed-partition-count", "1",
+                        "changed-partition-count", "0",
```

Comment:
the table is actually unpartitioned, so this should be 0

Reply:
I think it is reasonable to consider the root a changed partition.

Reply:
you're right. I've revisited the taken approach and updated it.